src/modules/endpoint/models/DateModel.js

```diff
@@ -24,6 +24,37 @@ class DateModel {
   get allDay (): boolean {
     return this._allDay
   }
+
+  /**
+   * Returns a formatted string containing all relevant start and end date and time information
+   * @param {String} locale The locale to format the span in
+   * @return {String} The formatted span string
+   */
+  toFormattedString (locale: string): string {
```

Comment:
@maxammann I would leave this in the model if it's okay. I like it better like this and think it makes things a little bit easier.

Reply:
Imo this is code to render a two timestamp and belongs next to a render method, but you can leave it here.

Reply:
@Schedulaar what do you think?

Reply:
I don't really mind 😅